source/validate.cpp

```diff
@@ -31,6 +31,10 @@
 #include <string.h>
 
 #include <vector>
+#include <iterator>
```

Comment:
We sort these. Not a big deal, we can come back to this.  The rest of the code is not fully compliant either.
For reference: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes